conf/fluo-deploy.props.example

```diff
@@ -54,6 +54,8 @@ default.instance.type = m3.large
 key.name = my_aws_key
 # Tags to add instances
 #instance.tags = key1:value1,key2:value2
+#Number of ephemeral drives to setup.  Do not set higher than supported by instance type.
+#ephemeral.num = 0
```

Comment:
Yes I think it should use max number of available ephemeral drives as you are already paying for them.